osu.Game.Tests/NonVisual/Filtering/FilterQueryParserTest.cs

```diff
@@ -268,10 +268,71 @@ public void TestApplyStatusQueries()
             FilterQueryParser.ApplyQueries(filterCriteria, query);
             Assert.AreEqual("I want the pp", filterCriteria.SearchText.Trim());
             Assert.AreEqual(4, filterCriteria.SearchTerms.Length);
-            Assert.AreEqual(BeatmapOnlineStatus.Ranked, filterCriteria.OnlineStatus.Min);
-            Assert.IsTrue(filterCriteria.OnlineStatus.IsLowerInclusive);
-            Assert.AreEqual(BeatmapOnlineStatus.Ranked, filterCriteria.OnlineStatus.Max);
-            Assert.IsTrue(filterCriteria.OnlineStatus.IsUpperInclusive);
+            Assert.IsNotEmpty(filterCriteria.OnlineStatus.Values);
+            Assert.That(filterCriteria.OnlineStatus.Values, Contains.Item(BeatmapOnlineStatus.Ranked));
+        }
+
+        [Test]
+        public void TestApplyMultipleEqualityStatusQueries()
+        {
+            const string query = "status=ranked status=loved";
+            var filterCriteria = new FilterCriteria();
+            FilterQueryParser.ApplyQueries(filterCriteria, query);
+            Assert.That(filterCriteria.OnlineStatus.Values, Is.Empty);
```

Comment:
This is no longer correct, from the conversations before it seemed like this should follow stable behavior and return loved beatmaps

Reply:
See above:

> This should ideally match nothing